app/controllers/v0/profile/payment_history_controller.rb

```diff
@@ -6,8 +6,9 @@ class PaymentHistoryController < ApplicationController
       before_action { authorize :bgs, :access? }
 
       def index
+        person = BGS::PeopleService.new(current_user).find_person_by_participant_id
         service = BGS::PaymentService.new(current_user)
```

Comment:
Since `person = BGS::PeopleService.new(current_user).find_person_by_participant_id` is a one-liner, maybe we should do that for this too:
```
service = BGS::PaymentService.new(current_user)
response = service.payment_history(person)
```

So it would be `response = BGS::PaymentService.new(current_user).payment_history(person)`

Reply:
Great idea!  Updated as suggested.